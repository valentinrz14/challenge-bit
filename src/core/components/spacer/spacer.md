```TSX
import React, { FunctionComponent } from 'react';
import { Spacer } from './spacer';

export const App: FunctionComponent = () => {

  return <Spacer height={10} />

};

```
