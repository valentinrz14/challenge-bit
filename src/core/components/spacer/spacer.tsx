import React, { FunctionComponent } from 'react';
import { View } from 'react-native';

/*
 ** Types
 */

interface SpacerProps {
  height: number;
}

/*
 ** Components
 */

export const Spacer: FunctionComponent<SpacerProps> = ({ height }) => {
  return <View style={{ height }} />;
};
