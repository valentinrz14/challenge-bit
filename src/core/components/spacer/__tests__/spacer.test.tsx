import React from 'react';
import { render } from '@testing-library/react-native';
import { Spacer } from '../spacer';

describe('Spacer', () => {
  it('should render correctly', () => {
    const { toJSON } = render(<Spacer height={10} />);
    expect(toJSON).toMatchSnapshot();
  });
});
