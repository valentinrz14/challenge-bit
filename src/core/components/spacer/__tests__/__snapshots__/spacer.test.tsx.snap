// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render correctly 1`] = `[Function]`;
